packages/remote-server/package.json

```diff
@@ -11,8 +11,10 @@
   "dependencies": {
     "@veramo/core": "^4.1.1",
     "@veramo/remote-client": "^4.1.1",
+    "@veramo/utils": "^4.1.1",
     "debug": "^4.3.3",
     "did-resolver": "^4.0.1",
+    "multibase": "^4.0.6",
```

Comment:
This does not seem to be in use anywhere, did I miss something?

Reply:
no you're right. was brought in accidentally before I refactored the util function out